packages/kbn-securitysolution-list-constants/src/index.ts

```diff
@@ -20,6 +20,11 @@ export const LIST_PRIVILEGES_URL = `${LIST_URL}/privileges`;
 export const EXCEPTION_LIST_URL = '/api/exception_lists';
 export const EXCEPTION_LIST_ITEM_URL = '/api/exception_lists/items';
 
+/**
+ * Internal exception list routes
+ */
+export const INTERNAL_EXCEPTION_LIST_URL = `/internal${EXCEPTION_LIST_URL}`;
```

Comment:
Just noticed further below that the actual route has `/_create` on it.  Can you instead define that here so that we don't have to hard coded everywhere we need it? (and still consider the name change if it makes sense)

Like:

```
/**
 * Ensure that a given list is created. If already created, then the existing list definition is returned (no 409 error). 
 * If not yet created, then it will be created.
 */
export const INTERNAL_EXCEPTIONS_LIST_ENSURE_CREATED_URL = `${INTERNAL_EXCEPTION_LIST_URL}/_ensure_created`;
```

Reply:
Would we consider not adding `_ensure...` if we foresee using this pattern for other routes - like an internal _update route that's for internally managed exception lists.